saleor/static/dashboard/js/components/google-preview.js

```diff
@@ -0,0 +1,66 @@
+const $seoTitle = $('#seo_title');
+
+const $nameId = $seoTitle.data('bind');
+const $name = $(`#${$nameId}`);
+
+const $seoDescription = $('#seo_description');
+const $descriptionMaterialize = $seoDescription.data('materialize');
+if ($descriptionMaterialize) {
+  var $description = $(`.materialize-textarea[name='${$descriptionMaterialize}']`);
```

Comment:
Don't use `var` - in ES6 you should use `let` in variable declaration and then change it's value